src/components/taskCreator.js

```diff
@@ -28,8 +30,8 @@ export default function TaskCreator({ createTask }) {
                 onChange={(e) => setTaskTitle(e.target.value)}
                 value={taskTitle} />
 
-            <button class="btn btn-secondary btn-add" id="add-task">
-                <span class="material-symbols-outlined" id="inputGroup-sizing-default">
+            <button className="btn btn-secondary btn-add" id="add-task">
```

Comment:
I would add in the button tag for this other tag:
```shell
<button className="btn btn-secondary btn-add" id="add-task" disabled={taskTitle.trim().length == 0}>
```
So it just disables the button when there's no input. It's more compelling, and it helps the user experience.

Reply:
Agreed, submitted the appropriate changes